### DISCUSSION TITLE
Search registry by date range

### DISCUSSION THREAD

**Discussion by Xiija** [npm/feedback#515] [Website] [answered]
I'd like to be able to check daily, weekly, or monthly for newly
published or modified packages.  Is there any api or info on how to do this, 
or could a date-range option be added to the website search?

**Comment by MylesBorins** [comment, marked as answer]
Hey @Xiija. At this time we are not offering additional ways to filter search results

We are tracking general improvements to search on our [public roadmap](https://github.com/npm/roadmap/projects/1#card-49987280), but we have not prioritized work in this area at this time.
